addons/pos_event/static/src/app/screens/product_screen/product_screen.js

```diff
@@ -0,0 +1,105 @@
+// Part of Odoo. See LICENSE file for full copyright and licensing details.
+import { ProductScreen } from "@point_of_sale/app/screens/product_screen/product_screen";
+import { makeAwaitable } from "@point_of_sale/app/store/make_awaitable_dialog";
+import { patch } from "@web/core/utils/patch";
+import { EventConfiguratorPopup } from "@pos_event/app/popup/event_configurator_popup/event_configurator_popup";
+import { _t } from "@web/core/l10n/translation";
+
+patch(ProductScreen.prototype, {
+    get products() {
+        const products = super.products;
+        const fakeEventProducts = this.pos.models["event.event"]
```

Comment:
I have no idea if related to this (maybe it's even on my side) but when searching for a string like 'erezrezrzer' in the POS, an error occurs. Still commenting if you have an idea?